desktop/apps/categories3/client.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './components/App'
+import { rehydrateClient } from 'desktop/components/react/utils/renderReactLayout'
+
+const bootstrapData = rehydrateClient(window.__BOOTSTRAP__)
+
+ReactDOM.render(
+  <App {...bootstrapData} />, document.getElementById('react-root')
+)
```

Comment:
This little block should be wrapped in an init function similar to how its done [here](https://github.com/artsy/force/blob/master/desktop/apps/react_example/client.js#L6); this is so asset packages are only loaded when needed.